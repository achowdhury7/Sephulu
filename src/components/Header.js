import React, { Component } from 'react'
import { Col, FormControl, FormGroup, Glyphicon, Grid, Row } from 'react-bootstrap'

const Header = (props) => {
  return (
    <Grid>
      <Row className="header-container">
        <Col md={4}>
          <div className="title">
            <h1>Sephulu</h1>
          </div>
        </Col>   
      </Row>
    </Grid>
  )
}

export default Header