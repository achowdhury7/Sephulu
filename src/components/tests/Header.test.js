import React from 'react'
import { shallow, mount, render } from 'enzyme'

import Header from '../Header'

test('render a header', () => {
  const wrapper = shallow(
    <Header />
  )
  expect(wrapper).toMatchSnapshot()
})