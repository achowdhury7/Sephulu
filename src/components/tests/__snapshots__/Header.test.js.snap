// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a header 1`] = `
<div
  className="header-container"
>
  <div
    className="title"
  >
    <h1>
      Sephulu
    </h1>
  </div>
</div>
`;
